translations/translations.json

```diff
@@ -1132,7 +1132,9 @@
       "back": "Zurück",
       "ok": "Ok",
       "minimize": "Minimieren",
-      "privacyPolicy": "Ich habe die <1>Datenschutzerklärung</1> gelesen und stimme der Verarbeitung und Speicherung meiner Daten zu."
+      "privacyPolicy": "Ich habe die <1>Datenschutzerklärung</1> gelesen und stimme der Verarbeitung und Speicherung meiner Daten zu.",
+      "accessible": "Barrierefrei",
+      "notAccessible": "Nicht barrierefrei"
```

Comment:
I guess we missed it here to add the constants for English as well, we should take care in the future to always also add new translations in English.

Reply:
Good point, thank you!